doc/intro.md

```diff
@@ -161,6 +161,13 @@ returning the results.
 FROM languages
 ```
 
+The `myvar= <<` supports flexible spaceing, such as `myvar = <<` and `myvar =<<`. All these syntaxs can work, for example:
```

Comment:
`spaceing` -> `spacing`
`syntaxs` -> `syntaxes`

Please reframe to :

`Flexible spacing is supported in assignments, e.g., `myvar = <<` and `myvar =<<`. Let's see an example below:`

```{code-cell} ipython3
%%sql lang = << SELECT *
FROM languages
```

Reply:
Use can use Grammarly for checking typos and grammatical errors.

Reply:
Got it, I will be more careful checking those typos and grammatical errors next time.